common.js

```diff
@@ -1943,6 +1943,74 @@ const allSoloCars = {
       '2012': ['gs']
     },
   },
+  'Infinti': {
+    'G35': {
```

Comment:
You should simply add `esp` to both the G35 coupe and sedan models you already have.

Reply:
And remove the `G35` model